backend/src/main/java/com/github/chuettenrauch/mixifyapi/mixtape_user/service/MixtapeUserService.java

```diff
@@ -19,11 +20,21 @@ public List<MixtapeUser> findAllByUser(User user) {
         return this.mixtapeUserRepository.findAllByUser(user);
     }
 
+    public MixtapeUser findByUserAndMixtape(User user, Mixtape mixtape) {
+        return this.mixtapeUserRepository
+                .findByUserAndMixtape(user, mixtape)
+                .orElseThrow(NotFoundException::new);
+    }
+
```

Comment:
Wozu brauchst du dir Methode?

Reply:
die wird beim `GET /api/mixtapes/{id}` benutzt. 
Dort war es ja vorher so, dass immer geguckt wurde, ob es ein Mixtape mit dem eingeloggten User als createdBy gibt und dieses zurückgegeben wird. 
Jetzt muss ich aber sicherstellen, dass man auch die Daten des Mixtapes bekommt, wenn man nicht der Creator is, aber das Mixtape mit einem geteilt wurde. 
Deswegen geh ich an der Stelle dann nicht mehr über die mixtape collection, sondern über die `mixtapeUser` collection, wo für das Mixtape alle User hinterlegt sind, die Zugriff haben.